content/docs/tasks/security/vault-ca/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Istio Vault CA integration
+description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+This tutorial walks you through an example to plug in a Vault CA for issuing
+certificates in Istio.
+
+## Before you begin
+
+* Create a new Kubernetes cluster to run the example in this tutorial.
+
+* On GKE, you may set the value of the `CLUSTER`, `ZONE`, and `PROJECT`
+environmental variables based on your GKE cluster and run the following
+`gcloud` command to fetch the credentials for your cluster.
+
+    {{< text bash >}}
+    $ export CLUSTER=YOUR-CLUSTER-NAME
+    $ export ZONE=YOUR-CLUSTER-ZONE
+    $ export PROJECT=YOUR-GKE-PROJECT-NAME
+    $ gcloud container clusters get-credentials $CLUSTER --zone $ZONE --project $PROJECT
+    {{< /text >}}
+
+## Install Istio with SDS enabled
+
+1.  Use `git clone` to download the latest Istio source code.
+
+    {{< text bash >}}
+    $ mkdir -p ~/go/src/istio.io
+    $ cd ~/go/src/istio.io
+    $ git clone https://github.com/istio/istio.git
+    $ cd istio
+    {{< /text >}}
+
+1.  Install Istio with SDS enabled and Node Agent sending certificate signing
+requests to a testing Vault CA (this example uses the latest release on
+2019-01-15, you may also use other releases):
+
+    {{< text bash >}}
+    $ export HUB=gcr.io/istio-release
+    $ export TAG=release-1.1-20190115-09-15
+    $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
+    $ helm dep update --skip-refresh install/kubernetes/helm/istio
+    $ ./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
+    $ cat install/kubernetes/namespace.yaml > istio-auth.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-* >> istio-auth.yaml
+    $ helm template --set global.tag=${TAG} \
+        --name=istio \
+        --namespace=istio-system \
+        --set global.hub=${HUB} \
+        --set global.mtls.enabled=true \
+        --set global.controlPlaneSecurityEnabled=true \
+        --set global.proxy.excludeIPRanges="35.233.249.249/32" \
+        --values install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml \
+        install/kubernetes/helm/istio >> istio-auth.yaml
+    $ kubectl create -f istio-auth.yaml
+    {{< /text >}}
+
+The testing Vault server used in this tutorial has the IP
+address `35.233.249.249`. The configuration
+`global.proxy.excludeIPRanges="35.233.249.249/32"` whitelists the IP address of
+the testing Vault server, so that Envoy will not intercept the traffic from
+Node Agent to Vault.
+
+The yaml file `install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml`
```

Comment:
Use `{{< github_file >}}install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml{{< /github_file >}}` which will produce a link to the file in GitHub.

Reply:
Done.